packages/@webex/contact-center/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@webex/contact-center",
+  "description": "",
+  "main": "dist/index.js",
+  "devMain": "src/index.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webex/webex-js-sdk.git",
+    "directory": "packages/@webex/contact-center"
+  },
+  "engines": {
+    "node": ">=20.x"
+  },
+  "scripts": {
+    "build:src": "webex-legacy-tools build -dest \"./dist\" -src \"./src\" -js -ts && yarn build",
+    "build": "yarn run -T tsc --declaration true --declarationDir ./dist/types",
+    "deploy:npm": "yarn npm publish",
+    "docs": "typedoc --emit none",
+    "fix:lint": "eslint 'src/**/*.ts' --fix",
+    "fix:prettier": "prettier \"src/**/*.ts\" --write",
+    "prebuild": "rimraf dist",
+    "test": "yarn test:style && yarn test:unit",
+    "test:style": "eslint 'src/**/*.ts'",
+    "test:unit": "webex-legacy-tools test --unit --runner jest"
+  },
+  "dependencies": {
+    "@types/platform": "1.3.4",
+    "@webex/contact-center": "workspace:*",
+    "@webex/internal-plugin-llm": "workspace:*",
+    "@webex/internal-plugin-mercury": "workspace:*",
+    "@webex/webex-core": "workspace:*",
+    "async-mutex": "0.4.0",
+    "buffer": "6.0.3",
+    "jest-html-reporters": "3.0.11",
+    "platform": "1.3.6",
+    "uuid": "8.3.2",
+    "xstate": "4.30.6"
+  },
+  "devDependencies": {
+    "@babel/preset-typescript": "7.22.11",
+    "@types/jest": "27.4.1",
+    "@typescript-eslint/eslint-plugin": "5.38.1",
+    "@typescript-eslint/parser": "5.38.1",
+    "@webex/babel-config-legacy": "workspace:*",
+    "@webex/eslint-config-legacy": "workspace:*",
+    "@webex/jest-config-legacy": "workspace:*",
+    "@webex/legacy-tools": "workspace:*",
+    "@webex/test-helper-mock-webex": "workspace:*",
+    "eslint": "^8.24.0",
+    "eslint-config-airbnb-base": "15.0.0",
+    "eslint-config-prettier": "8.3.0",
+    "eslint-import-resolver-typescript": "2.4.0",
+    "eslint-plugin-import": "2.25.3",
+    "eslint-plugin-jsdoc": "38.0.4",
+    "eslint-plugin-prettier": "4.0.0",
+    "eslint-plugin-tsdoc": "0.2.14",
+    "jest": "27.5.1",
+    "jest-junit": "13.0.0",
+    "jsdom": "19.0.0",
+    "jsdom-global": "3.0.2",
+    "prettier": "2.5.1",
+    "sinon": "^9.2.4",
```

Comment:
do we need deps like jsdom, sinon etc?

Reply:
jsdom might be required because we will have this running on the browser so if there is use of a window object, we will need jsdom. Sinon might be necessary if we need to use the same testing logic of stubbing functions

Reply:
removed the extra dependencies